File: EJERCICIOS DE LOS ALUMNOS/JAVI/Python/7º Ejercicio/edad.py
edad = int(input("Dime tu edad: "))

if edad >= 18:
    print("Eres mayor de edad")
else:
    if edad > 12 and edad < 18:
        print("Eres un adolescente")
    else:
        print("Eres un pipiolo")
        

File: EJERCICIOS DE LOS ALUMNOS/JAVI/JAVA/ProgSistemas/src/Metodos/Edad.java
package Metodos;

public class Edad {
    
    
    public int nac;
    
    
    public int calcularEdad() {
        return 2026 - nac;
    }
    
    
    public String mayorEdad() {
        
        if (calcularEdad() >= 18) {
            return "Eres mayor de edad.";
        } else {
            return "NO eres mayor de edad.";
        }
    }
    
    public void resultados() {
        System.out.println("Hola, tu edad es de: " + calcularEdad());
        
        System.out.println(mayorEdad());
    }
}